README.md

```diff
@@ -134,6 +134,21 @@ Analyzer is installed. For example, on Windows the default installation
 location is C:\\Program
 Files\\Fortify\\Fortify\_SCA\_and\_Apps\_*\<version\>*.
 
+### Configuration when running Jenkins from the Docker
+
+When running Jenkins from under the Docker container you need to mount *\<sca\_install\_dir\>* 
+directory to the Docker container in order to make Fortify Static Code Analyzer executables
+accessible from under the Docker. For instance, next command does this:
+
+    docker container run \
+      -p 8080:8080 \
+      -v /home/admin/Fortify/Fortify_SCA_and_Apps_20.2.0:/var/jenkins_home/Fortify/Fortify_SCA_and_Apps_20.2.0 \
+      --name=jenkins \
+      jenkins/jenkins -d
+
+The corresponding FORTIFY\_HOME should have the value `/var/jenkins_home/Fortify/Fortify_SCA_and_Apps_20.2.0` 
```

Comment:
-> For this example, the value of FORTIFY\_HOME is ‘/var/Jenkins_home/Fortify/Fortify_SCA_and_Apps_20.2.0’.

Reply:
Done.